pages/services/kafka/2.6.0-2.2.1/started/index.md

```diff
@@ -0,0 +1,617 @@
+1. The presence (or not) of a SIGKILL log line indicating that the underlying Kafka broker did not shutdown cleanly within the
+   allotted grace period.
+1. The task status update marked by `TASK_KILLED`, indicating the end of the shutdown activity.
+
+## Broker Count
+
+Configure the number of brokers running in a given Kafka cluster. The default count at installation is three brokers. This number may be increased, but not decreased, after installation.
+
+*   **In DC/OS CLI options.json**: `broker-count`: integer (default: `3`)
+*   **DC/OS web interface**: `BROKER_COUNT`: `integer`
+
+## Broker Port
+
+Configure the port number that the brokers listen on. If the port is set to a particular value, this will be the port used by all brokers. The default port is 9092.  Note that this requires that `placement-strategy` be set to `NODE` to take effect, since having every broker listening on the same port requires that they be placed on different hosts. Setting the port to 0 indicates that each Broker should have a random port in the 9092-10092 range.
+
+*   **In DC/OS CLI options.json**: `broker-port`: integer (default: `9092`)
+*   **DC/OS web interface**: `BROKER_PORT`: `integer`
+
+## Configure Broker Placement Strategy <!-- replace this with a discussion of PLACEMENT_CONSTRAINTS? -->
+
+`ANY` allows brokers to be placed on any node with sufficient resources, while `NODE` ensures that all brokers within a given Kafka cluster are never colocated on the same node. This is an option that cannot be changed once the Kafka cluster is started: it can only be configured via the DC/OS CLI `--options` flag when the Kafka instance is created.
+
+*   **In DC/OS CLI options.json**: `placement-strategy`: `ANY` or `NODE` (default: `ANY`)
+*   **DC/OS web interface**: `PLACEMENT_STRATEGY`: `ANY` or `NODE`
+
+## Configure Kafka Broker Properties
+
+Kafka Brokers are configured through settings in a server.properties file deployed with each Broker. The settings here can be specified at installation time or during a post-deployment configuration update. They are set in the DC/OS Universe's config.json as options such as:
+
+```json
+    "log_retention_hours": {
+        "title": "log.retention.hours",
+        "description": "Override log.retention.hours: The number of hours to keep a log file before deleting it (in hours), tertiary to log.retention.ms property",
+        "type": "integer",
+        "default": 168
+    },
+```
+
+The defaults can be overridden at install time by specifying an options.json file with a format like this:
+
+```json
+    {
+        "kafka": {
+            "log_retention_hours": 100
+        }
+    }
+```
+
+These same values are also represented as environment variables for the scheduler in the form `KAFKA_OVERRIDE_LOG_RETENTION_HOURS` and may be modified through the DC/OS web interface and deployed during a rolling upgrade as [described here][12].
```

Comment:
Replace `described here[12]` with `described in [changing configuration at runtime](https://docs.mesosphere.com/services/kafka/2.6.0-2.2.1/started/#changing-configuration-at-runtime)